fms2_io/include/register_domain_restart_variable.inc

```diff
@@ -25,31 +25,47 @@
 
 !> @brief Add a domain decomposed variable.
 subroutine register_domain_restart_variable_0d(fileobj, variable_name, vdata, &
-                                               dimensions, is_optional)
+                                               dimensions, is_optional, deflate_level, &
+                                               chunksizes)
 
   type(FmsNetcdfDomainFile_t), intent(inout) :: fileobj !< File object.
   character(len=*), intent(in) :: variable_name !< Variable name.
   class(*),  intent(in), target :: vdata !< Variable data.
   character(len=*), dimension(:), intent(in), optional :: dimensions !< Dimension names.
   logical, intent(in), optional :: is_optional !< Prevent errors in read-only files
                                                !! if a variable does not exist.
-
-  call netcdf_add_restart_variable(fileobj, variable_name, vdata, dimensions, is_optional)
+  integer, intent(in), optional :: deflate_level !< The netcdf deflate level
+                                                 !! This feature is only
+                                                 !! available for netcdf4 files
+  integer, intent(in), optional :: chunksizes(:) !< netcdf chunksize to use for this variable
+                                                 !! This feature is only
+                                                 !! available for netcdf4 files
+
+  call netcdf_add_restart_variable(fileobj, variable_name, vdata, dimensions, &
```

Comment:
also just needs is_optional added to this call. Applies to the rest of these routines as well

Reply:
@mcallic2 This file is still missing the `is_optional`